src/natcap/invest/habitat_quality.py

```diff
@@ -118,29 +118,33 @@
                     "type": "raster",
                     "bands": {1: {"type": "ratio"}},
                     "about": gettext(
-                        "Map of the threat's distribution in the current "
-                        "scenario. Each pixel value is the relative intensity "
-                        "of the threat at that location. ")
+                        "File system path to a raster of the threat's "
```

Comment:
I would suggest leaving out the "File system" from each of these. Users may not know what a file system is, and it might just confuse them.